geomstats/geometry/special_orthogonal.py

```diff
@@ -138,6 +138,34 @@ def random_uniform(self, n_samples=1, tol=1e-6):
         skew = self.to_tangent(random_mat)
         return self.exp(skew)
 
+    @geomstats.vectorization.decorator(['else', 'vector'])
```

Comment:
This method is not tested

Reply:
I split the original method, which contained three cases, in each of the corresponding classes. Wasn't this part tested in test_special_euclidian already ?